exetera/core/fields.py

```diff
@@ -199,11 +224,14 @@ def complete(self):
                               "for a writeable copy of the field")
 
 
-# Field arrays
-# ============
-
 class WriteableFieldArray:
     def __init__(self, field, dataset_name):
+        """
+        Construct a read/write FieldArray which used as the wrapper of data in Field.
```

Comment:
Construct a WriteableFieldArray instance. This class is an implementation detail, used to access data in non-indexed fields deriving from HDF5Field. As such, instances of WriteableFieldArray should only be created by the Fields themselves, and not by users of ExeTera.